pkg/security/mcs/label.go

```diff
@@ -0,0 +1,178 @@
+package mcs
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const maxCategories = 1024
+
+type Labels struct {
```

Comment:
@pweil- @smarterclayton I notice we have carried over the label/level disconnect into this code.  In upstream, the field is called 'Label'.  In the actual SELinux docs, both terms are used.